Solutions/RubrikSecurityCloud/Data/Solution_RubrikSecurityCloud.json

```diff
@@ -15,10 +15,10 @@
         "Playbooks/RubrikRansomwareDiscoveryAndVMRecovery/azuredeploy.json"
     ],
     "Data Connectors": [
-        "Data Connectors/RubrikWebhookEvents_API_FunctionApp.json"
+        "Data Connectors/RubrikDataConnector/Rubrik_FunctionApp.json"
```

Comment:
Hi @jayeshprajapaticrest, we can't find this file on the given path. And so we are getting this error while trying to repackage it. Please update the correct update and repackage the solution. 

![image](https://github.com/Azure/Azure-Sentinel/assets/120547590/5db1a9e4-314e-4950-ae42-164de4bdab0d)

Reply:
@v-rbajaj Updated the PR with the mentioned changes.

Reply:
Hi @jayeshprajapaticrest, as mentioned earlier, the solution needs to be repackaged too.

Reply:
@v-rbajaj I have make the folder path as mentioned in the above path and also used the same path in the SolutionMetadata.json file and re-packaged the solution. Please verify and review the PR.
Thanks